src/trade.js

```diff
@@ -10,6 +10,7 @@ class Trade {
     this.amount = props.amount;
     this.buyOrderId = props.buyOrderId;
     this.sellOrderId = props.sellOrderId;
+    this.sequenceId = props.sequenceId;
```

Comment:
minor: you can just remove this since the catch all below will ensure it is included if the argument object includes it.